tests/core/core.spec.ts

```diff
@@ -7,19 +7,27 @@ import { expect } from "chai";
 import { asPlainObject } from "df/tests/utils";
 import * as path from "path";
 
-const TEST_CONFIG: dataform.IProjectConfig = {
-  warehouse: "redshift",
-  defaultSchema: "schema"
-};
-
-const TEST_CONFIG_WITH_SUFFIX: dataform.IProjectConfig = {
-  ...TEST_CONFIG,
-  schemaSuffix: "suffix"
-};
+class TEST_CONFIG {
```

Comment:
I'd call this `TestConfigs` (we have classes as CamelCase, we should add a tslint rule for this!).

Reply:
Thanks, much prettier now